modules/ROOT/pages/_partials/secrets-manager.adoc

```diff
@@ -0,0 +1,713 @@
+
+Besides the default flag `--help`, this command also accepts:
+
+[%header,cols="30a,40a,30a"]
+|===
+|Flag | Description |  Example
+|`--group-id`| Mandatory. Secret group ID. | `--group-id 030493`
+|`--name`|Mandatory. Name for your secret. | `--name TestSecret`
+|`--max-tls-version` |Maximum TLS Version +
+Default: TLSv1.3. | `--tls-version TLSv1.1`
+|`--min-tls-version` |Minimum TLS Version +
+Default: TLSv1.1. | `--tls-version TLSv1.2`
+|`--cipher` |Cipher for the specified TLS version range. | `--cipher TLS_ECDHE_PSK_WITH_CHACHA20_POLY1305_SHA256`
+|`--alpn-protocol` |ALPN Protocol. +
+Options: h2, http/1.1 | `--alpn-protocol h2`
+|`--enable-client-cert-validation`| Enable client certificate validation | `--enable-client-cert-validation`
+|`--expiration-date`| Expiration date for the secret| `--expiration-date 01/01/2025`
+|`--keystore-id`|A valid PEM keystore ID in the secret group, which is used as keystore for the TLS context. | `--keystore-id 00003443`
+|`--truststore-id`|A valid PEM truststore ID in the secret group, which is used as truststore for the TLS context. | `--truststore-id 001111032`
+|`--skip-server-cert-validation` | Skip service certificate validation | `--skip-server-cert-validation`
+
+|===
+
+For more information on ciphers, please see xref:flex-gateway::conn-tls-config[Configuring TLS Context for Flex Gateway in Connected Mode]
+
+
+[[secret-TLS-flex-describe]]
+== secrets-mgr:tls-context:flex-gateway:describe
+
+----
+> secrets-mgr:tls-context:flex-gateway:describe [flags] 
+----
+
+Returns the details of a Flex Gateway TLS Context secret specified in `--id` from the secret group you pass in `--group-id`. 
+
+This command accepts the default flag `--help`.
+
+NOTE: The output does not include any sensitive or secret data.
+
+
+[[secret-TLS-flex-list]]
+== secrets-mgr:tls-context:flex-gateway:list
+
+----
+> secrets-mgr:tls-context:flex-gateway:list [flags]
+----
+
+Lists all Flex Gateway TLS Context secrets in a secret group specified in `--group-id`
+
+This command accepts the default flag `--help`.
+
+[[secret-TLS-flex-modify]]
+== secrets-mgr:tls-context:flex-gateway:modify
+
+----
+> secrets-mgr:TLS-context:flex-gateway:modify  [flags] 
+----
+
+Modifies the name or expiration date for a Flex Gateway TLS Context secret specified in `--id` from the group you pass in`--groupid`. 
+
+Besides the default flag `--help`, this command also accepts:
+
+[%header,cols="30a,40a,30a"]
+|===
+|Flag | Description |  Example
+|`--name` | New name for the truststore secret | `--name TestTruststoreSecret`
+|`--expiration-date` | New expiration date for the truststore secret | `--expiration-date 01/01/2025`
+
+|===
+
+[[secrets-TLS-flex-replace]]
+== secrets-mgr:tls-context:flex-gateway:replace
+
+----
+> secrets-mgr:tls:context:flex-gateway:replace  [flags] 
+----
+
+Replaces an existing Flex Gateway TLS Context secret you pass in `--id`, from the secret group you pass in `--group-id`, using the type you pass in `--type`. 
```

Comment:
For consistency, it should be
Replaces an existing Flex Gateway TLS Context secret **specified**  in `--id`...